src/cronGenerator.ts

```diff
@@ -0,0 +1,33 @@
+import cronstrue from 'cronstrue';
+
+export function generateCronExpression() {
+    function getRandomOrWildcard(min: number, max: number) {
+        if (Math.random() < 0.7) {
```

Comment:
It seems like `Math.random` cannot be seeded, so we can't really unit test this part of code. What we can do, however, is use [property based testing](https://github.com/dubzzz/fast-check) to fuzzy test the generator and verify if the output is always valid.

We can check validity by passing the output to `generadeReadableExpression` and expect it to NOT throw.